src/types.ts

```diff
@@ -237,4 +241,13 @@ export interface SettingsStorage {
   userEmail: string;
   locale: string;
   theme: string;
+  currency: SupportedCurrencies;
+  exchange: SupportedExchanges;
 }
+
+export type SupportedExchanges = "coindesk" | "yadio";
+
+// Supported currencies by Coindesk and yadio
+// FYI: yadio is i.e. not supporting "ISK", maybe more?
+// https://github.com/AryanJ-NYC/bitcoin-conversion/blob/master/src/index.ts#L143
+export type SupportedCurrencies = typeof CURRENCIES[number];
```

Comment:
@dylancom should `currencies` and `exchanges` somehow be `enums`?

Reply:
https://www.vendure.io/docs/typescript-api/common/currency-code/

Reply:
Seems like a practice.